### ISSUE TITLE
A post to demonstrate Pandoc's Markdown and bookdown's Markdown extensions

### ISSUE THREAD

**Issue by yihui** [apreshill/blogdown-demo#5]
We need some more examples to demonstrate Markdown features mentioned in Chapter 2 of the bookdown book: https://bookdown.org/yihui/bookdown/components.html So far we have only covered citations and HTML widgets.

**Comment by apreshill** [comment]
Outline here for me to tackle tomorrow:

1. R code [see pull request #9 ]

1. Figures and other images [see pull request #10 ]

1. Tables [see pull request #15  ]

1. Cross-references

1. Citations [see pull request #8 ]

1. HTML widgets [see pull request #6 ]

1. Web pages and shiny apps

Not applicable/wait on:

1. Index

1. Custom blocks

When done:

- [ ] Update about.md with new post blurbs

**Comment by yihui** [comment]
Yeah, that sounds perfect. Custom blocks is not necessary. If you don't have time, you can omit it, since it does not work out of the box -- it requires you to write some additional CSS rules.
